src/odemis/driver/zeiss.py

```diff
@@ -598,9 +598,11 @@ def _doMoveRel(self, future, shift):
         if "rx" in shift:
             move_list[3] += shift["rx"]
             target_pos["rx"] = move_list[3]
-        if "rz" in shift:
-            move_list[4] += shift["rz"]
-            target_pos["rz"] = move_list[4]
+        if "rm" in shift:
+            move_list[4] += shift["rm"]
+            move_list[4] = move_list[4] % (2 * math.pi)
+            # correct for full rotation under and overflow
+            target_pos["rm"] = move_list[4] % (2 * math.pi)
```

Comment:
Though the result is same, just easy to read
```suggestion
            target_pos["rm"] = move_list[4] 
```

Reply:
Yeah good point that actually a remainder of quick programming where the second use of (2 * math.pi) is redundant.
Changed it